document/core/exec/instructions.rst

```diff
@@ -381,7 +381,7 @@ Memory Instructions
 ~~~~~~~~~~~~~~~~~~~
 
 .. note::
-   The alignment :math:`\memarg.\ALIGN` in load and store instructions does not affect the semantics.
+   The alignment :math:`\memarg.\ALIGN` in load and store instructions does not affect the semantics of accesses to :ref:`unshared <syntax-unshared>` :ref:`memories <syntax-mem>`.
```

Comment:
Even for shared memories, non-atomic accesses are allowed to have incorrect alignment hints, I think?